saleor/account/migrations/0040_auto_20200415_0443.py

```diff
@@ -0,0 +1,32 @@
+# Generated by Django 3.0.5 on 2020-04-15 09:43
+
+from django.db import migrations
+
+
+def change_extension_permission_to_plugin_permission(apps, schema_editor):
+    permission = apps.get_model("auth", "Permission")
+    users = apps.get_model("account", "User")
+    plugin_permission = permission.objects.get(
```

Comment:
I think this `get()` and the one below may be the reason for failing CI now. Instead maybe you should use `filter()[0]` with some null check; then based on that run the right logic.

Reply:
`filter` then `first`

Reply:
Yup that what I actually meant ;)